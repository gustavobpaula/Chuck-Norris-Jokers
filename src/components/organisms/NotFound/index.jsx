import React from 'react';
import notFound from '../../../assets/images/404.gif';
import { Image, CenterMiddle } from '../../atoms';

const NotFound = () => (
  <CenterMiddle>
    <Image src={notFound} alt="404 - Not Found" />
  </CenterMiddle>
);

export default NotFound;
