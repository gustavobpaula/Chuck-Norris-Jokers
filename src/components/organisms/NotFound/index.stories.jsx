import React from 'react';
import NotFound from './index';

export default {
  component: NotFound,
  title: 'Organisms/NotFound',
};

export const element = () => <NotFound />;

element.story = {
  name: 'Default',
};
