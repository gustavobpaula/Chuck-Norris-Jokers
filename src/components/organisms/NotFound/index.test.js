import React from 'react';
import { mount } from 'enzyme';
import NotFound from './index';

describe('<NotFound />', () => {
  it('should mount properly', async () => {
    const wrapper = mount(<NotFound />);

    expect(wrapper).toMatchSnapshot();
  });
});
