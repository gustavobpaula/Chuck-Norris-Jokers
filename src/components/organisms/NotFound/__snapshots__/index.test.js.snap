// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should mount properly 1`] = `
<NotFound>
  <CenterMiddle>
    <style__Center>
      <StyledComponent
        forwardedComponent={
          Object {
            "$$typeof": Symbol(react.forward_ref),
            "attrs": Array [],
            "componentStyle": ComponentStyle {
              "componentId": "style__Center-sc-1qm54d6-0",
              "isStatic": false,
              "lastClassName": "hWvMyz",
              "rules": Array [
                "left:50%;position:absolute;top:50%;transform:translate(-50%,-50%);",
              ],
            },
            "displayName": "style__Center",
            "foldedComponentIds": Array [],
            "render": [Function],
            "styledComponentId": "style__Center-sc-1qm54d6-0",
            "target": "div",
            "toString": [Function],
            "warnTooManyClasses": [Function],
            "withComponent": [Function],
          }
        }
        forwardedRef={null}
      >
        <div
          className="style__Center-sc-1qm54d6-0 hWvMyz"
        >
          <Image
            alt="404 - Not Found"
            height="auto"
            maxHeight="100%"
            maxWidth="100%"
            src="test-file-stub"
            width="auto"
          >
            <style__Image
              alt="404 - Not Found"
              height="auto"
              maxHeight="100%"
              maxWidth="100%"
              src="test-file-stub"
              width="auto"
            >
              <StyledComponent
                alt="404 - Not Found"
                forwardedComponent={
                  Object {
                    "$$typeof": Symbol(react.forward_ref),
                    "attrs": Array [],
                    "componentStyle": ComponentStyle {
                      "componentId": "style__Image-zsbil-0",
                      "isStatic": false,
                      "lastClassName": "bSPXyq",
                      "rules": Array [
                        "height:",
                        [Function],
                        ";max-height:",
                        [Function],
                        ";max-width:",
                        [Function],
                        ";width:",
                        [Function],
                        ";",
                      ],
                    },
                    "displayName": "style__Image",
                    "foldedComponentIds": Array [],
                    "render": [Function],
                    "styledComponentId": "style__Image-zsbil-0",
                    "target": "img",
                    "toString": [Function],
                    "warnTooManyClasses": [Function],
                    "withComponent": [Function],
                  }
                }
                forwardedRef={null}
                height="auto"
                maxHeight="100%"
                maxWidth="100%"
                src="test-file-stub"
                width="auto"
              >
                <img
                  alt="404 - Not Found"
                  className="style__Image-zsbil-0 bSPXyq"
                  height="auto"
                  src="test-file-stub"
                  width="auto"
                />
              </StyledComponent>
            </style__Image>
          </Image>
        </div>
      </StyledComponent>
    </style__Center>
  </CenterMiddle>
</NotFound>
`;
